app/views/publications/mid_cycle_report/_table_body.html.erb

```diff
@@ -0,0 +1,28 @@
+<thead class="govuk-table__head">
+  <tr class="govuk-table__row">
+    <th scope="col" class="govuk-table__header">Level</th>
+    <th scope="col" class="govuk-table__header">This cycle</th>
+    <th scope="col" class="govuk-table__header">Last cycle</th>
+    <th scope="col" class="govuk-table__header">Percentage change</th>
+  </tr>
+</thead>
+<tbody class="govuk-table__body">
+  <% model.each do |row| %>
+    <tr class="govuk-table__row">
+      <th scope="row" class="govuk-table__header govuk-!-font-weight-regular"><%= row['tad_subject'] %></th>
+      <td class="govuk-table__cell"><%= row[first_column] %></td>
+      <td class="govuk-table__cell"><%= row[second_column] %></td>
+      <td class="govuk-table__cell"><%= row[third_column] %></td>
```

Comment:
Adding this extra class makes the numbers right-aligned and using a fixed-width font. This makes the numbers easier to compare visually.

```suggestion
      <td class="govuk-table__cell govuk-table__cell--numeric"><%= row[first_column] %></td>
      <td class="govuk-table__cell govuk-table__cell--numeric"><%= row[second_column] %></td>
      <td class="govuk-table__cell govuk-table__cell--numeric"><%= row[third_column] %></td>
```